### PR TITLE
updated aja-system-test (2.1)

### PR DESCRIPTION
**Pull request by timsutton**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

**Comment by timsutton** [comment]
Just to clarify, the existing cask was downloading an older version (10.5.2) - this change fetches 12.3.7, which confusingly goes under the marketing version of "2.1".

File: Casks/aja-system-test.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cask 'aja-system-test' do
-  version :latest
-  sha256 :no_check
+  version '2.1'
```
Comment (vitorgalvao) [inline Casks/aja-system-test.rb]: `2.1.0`

File: Casks/aja-system-test.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+  sha256 '53fa38382d73a0e70a88f5f23fdb384612fd57dd8ae1bb28fe49a7896b321bac'
 
-  url 'https://www.aja.com/assets/products/files/90/AJASystemTestMac.zip'
+  url 'https://www.aja.com/assets/products/files/90/AJA_SystemTest-2.1.0.app.zip'
```
Comment (vitorgalvao) [inline Casks/aja-system-test.rb]: `url "https://www.aja.com/assets/products/files/90/AJA_SystemTest-#{version}.app.zip"`

**Comment by timsutton** [comment]
Thanks, just added that change!

**Pull request merged**